ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__cr-4.17-nightly.yaml

```diff
@@ -24,7 +24,7 @@ resources:
       memory: 200Mi
 tests:
 - as: krkn-hub-aws
-  cron: 0 3 * * 1,4
+  cron: 0 0 * * *
```

Comment:
think for 4.17 we can also lessen the cadence since its 2 versions behind

Reply:
From a component readiness perspective, typically the 'base' signal is gathered during the last 30 days of the dev period prior to GA.  So lowering the cadence would be fine.  Since you are just starting these jobs and collecting data it all depends on what releases you want to start leveraging CR for.  If it is 4.19 then getting enough data for just 4.18 would be fine.  If you also plan to monitor 4.18 then you likely want good base data from 4.17 as well.

Reply:
Makes sense @neisw! We definitely want to track regressions for 4.18 as well in addition to 4.19. Will go with a daily cadence for 4.17, 4.18 and 4.19 for now and tweak it based on the new releases. Thanks.

Reply:
@paigerube14 Thoughts?